### PR TITLE
[#7] Integrate a logging library

### PR DESCRIPTION
**Pull request by championballer**
#7

Integrated logging library logrus. Please tell me if any changes are required.

### CODE REVIEW

File: cmd/version.go [hunk 1]
```diff
@@ -15,6 +14,6 @@ var versionCmd = &cobra.Command{
 	Short: "Print the version number of Dunner",
 	Long:  `All software has versions. This is Dunners's`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("v0.1")
+		log.Println("v0.1")
```
Comment (agentmilindu) [inline cmd/version.go]: This makes the output to be `INFO[0000] v0.1` which is not nice, Shall we keep these as it is?

File: src/services/ConfigService/ConfigService.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package ConfigService
 import (
 	"log"
 	"io/ioutil"
+	log "github.com/sirupsen/logrus"
```
Comment (agentmilindu) [inline src/services/ConfigService/ConfigService.go]: If you rename `logrus` to `log` here, you have to remove the first imported `log`

File: cmd/root.go [hunk 3]
```diff
@@ -15,16 +15,16 @@ var rootCmd = &cobra.Command{
 
 		_, err := docker.NewEnvClient()
 		if err != nil {
-			panic(err)
+			log.Fatal(err)
 		}
 
-		fmt.Println("Dunner running!")
+		log.Println("Dunner running!")
```
Comment (agentmilindu) [inline cmd/root.go]: This makes the output to be `INFO[0000] Dunner running!` which is not nice, Shall we keep these as it is?

File: cmd/root.go [hunk 4]
```diff
@@ -15,16 +15,16 @@ var rootCmd = &cobra.Command{
...
 	},
 }
 
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(err)
+		log.Println(err)
```
Comment (agentmilindu) [inline cmd/root.go]: should be `log.Fatal()`

File: src/services/ConfigService/ConfigService.go (hunk 2 above)
Comment (championballer) [reply src/services/ConfigService/ConfigService.go]: Yeah, forgot. to remove this, will do. Thanks

File: cmd/root.go (hunk 4 above)
Comment (championballer) [reply cmd/root.go]: True. Will change.

File: cmd/version.go (hunk 1 above)
Comment (championballer) [reply cmd/version.go]: Alright.

File: cmd/root.go (hunk 3 above)
Comment (championballer) [reply cmd/root.go]: Alright.

**Comment by rehrumesh** [comment]
LGTM. Great work @championballer

**Pull request merged**